### PR TITLE
Repeat summary

### PR DESCRIPTION
**Pull request by georgehrke**
This implements a repeat summary and based on that a read-only view for the repeat tab.

60CE4628-359A-4B68-AA1F-CB4E833367C2

### CODE REVIEW

File: src/components/Editor/Repeat/RepeatSummary.vue [hunk 1]
```diff
@@ -21,14 +21,270 @@
...
+				byDayNames.push(allByDayNames[5])
+			}
+			if (byDayList.includes('SA')) {
+				byDayNames.push(allByDayNames[6])
+			}
+			if (byDayList.includes('SU')) {
+				byDayNames.push(allByDayNames[0])
+			}
+
+			return byDayNames.join(', ')
```
Comment (tcitworld) [inline src/components/Editor/Repeat/RepeatSummary.vue]: Support is pretty bad right now but we could use [`Intl.ListFormat.format()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ListFormat) here.

Comment (georgehrke) [reply src/components/Editor/Repeat/RepeatSummary.vue]: https://caniuse.com/#feat=internationalization

Support seems not that bad anymore

Comment (tcitworld) [reply src/components/Editor/Repeat/RepeatSummary.vue]: I feel like this caniuse page is generic and concerns things like `Intl.DateTimeFormat` (for which support is good).

Comment (georgehrke) [reply src/components/Editor/Repeat/RepeatSummary.vue]: Ah, yes.
MDN shows way more red: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ListFormat#Browser_compatibility

**Pull request merged**